catalog/spatial/registry/registry-admin-modules/registry-admin-local-ui/src/main/webapp/js/model/Field.js

```diff
@@ -111,7 +111,7 @@ define([
                     return;
                 }
             } else {
-                if (this.isEmpty() && !this.get('regex') && !this.get('required')) {
+                if (!this.get('value') && !this.get('regex') && !this.get('required')) {
```

Comment:
@andrewkfiedler, not sure what you mean by regex, but I think that the value could be any of the fields in the `Add Local Node` modal. The value being 0 could very well have been an issue for the lat and lon fields, so maybe I can write up another ticket for that. What do you think?